streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -72,6 +70,7 @@
     private boolean transactionInFlight = false;
     private final Time time;
     private final TaskMetrics metrics;
+    private final Logger log;
```

Comment:
Just use the one created in `AbstractTask`?

Reply:
Fixed